Packs/Gmail/Integrations/Gmail/README.md

```diff
@@ -6,56 +6,39 @@ This integration replaces the Gmail functionality in the GoogleApps API and G Su
 
 There are several procedures you have to perform in Google before configuring the integration on Demisto.
 
-* [Get a New Private Key](#get-a-new-private-key)
-* [Delegate Domain-wide Authority to Your Service Account](#delegate-domain-wide-authority-to-your-service-account)
-* [Get an Immutable Google Apps ID](#get-an-immutable-google-apps-id-parameters)
+* Get a New Private Key
+* Delegate Domain-wide Authority to Your Service Account
+* Get an Immutable Google Apps ID
```

Comment:
why did you remove the links?

Reply:
missed that while merging the conflict. i'll revert it.